webpack-test/checkArrayExpectation.js

```diff
@@ -82,6 +82,10 @@ module.exports = function checkArrayExpectation(
 		}
 	}
 	if (fs.existsSync(path.join(testDirectory, `${filename}.js`))) {
+		if (fs.existsSync(path.join(testDirectory, `${filename}-sort.js`))) {
```

Comment:
Add an optional sorter file, to sort `stats.errors`,  the order of `stats.errors` is hard to keep the same as webpack because our parallel architecture. And the order of `stats.errors` does not affect the correctness.